install.md

```diff
@@ -22,23 +22,11 @@ terminal interface.
 <summary>
 Windows
 </summary>
-We currently only support Windows 10 or newer using <a href="https://learn.microsoft.com/en-us/windows/wsl/install">Windows Subsystem for Linux (WSL)</a>.
+Please install Windows Subsystem for Linux (WSL) following the <a href="https://learn.microsoft.com/en-us/windows/wsl/install">official instructions</a>. You should now have an App "Ubuntu" in your start menu and a folder "Linux" on the left sidebar of your file explorer. You can now follow the instructions for <em><a href="#install-ubuntu">Ubuntu</a></em>. Please note: 
 <ol>
-<li>Search for "Turn Windows features on or off"</li>
-<li>On the left panel, select "Turn Windows features on or off"</li>
-<li>Select "Windows subsystem for Linux" and press "Ok"</li>
-<li>Click "Restart the PC"</li>
-<li>Click the Windows store icon (shopping bag)</li>
-<li>Search for "Ubuntu" in the store - it's free!</li>
-<li>Select "Ubuntu" and "Get" the app</li>
-<li>Click "Launch" and follow the prompts</li>
+<li> When you start Ubuntu, your terminal will be in the folder "Linux/Ubuntu/home/your_username>". Please avoid changing anything outside that folder unless you are absolutely sure.</li>
```

Comment:
I added the information so that the average user knows how to access the directory from Windows (so that they can download script or email their code). But I agree that telling users to simply type `explorer.exe .` is the easier option and we should add that instead of the actual path.